site/class-1-load/tpulido/index.csv

```diff
@@ -0,0 +1,4 @@
+day,dinner
+2017-09-25, Pasta bolognese
```

Comment:
CSV doesn’t allow padding around the commas, so in `,·`, the `·` is an actual space in the data.